server/app.js

```diff
@@ -0,0 +1,33 @@
+require('dotenv').config()
+const express = require('express')
+const app = express()
+const PORT = process.env.PORT
+const routes = require('./routes')
+const mongoose = require('mongoose');
+const cors = require('cors')
+const kue = require('kue')
+
+app.use(cors())
+mongoose.set('useFindAndModify', false);
+// mongoose.connect('mongodb://localhost/hacktivoverflow', { useNewUrlParser: true })
+mongoose.connect('mongodb+srv://admin:root@cluster0-r5gmo.gcp.mongodb.net/hijarvis?retryWrites=true', {useNewUrlParser: true})
```

Comment:
simpan dalam env

Reply:
selesai

env nya:
DB_DEV=mongodb://localhost/hacktivoverflow
```suggestion
mongoose.connect('mongodb+srv://admin:root@cluster0-r5gmo.gcp.mongodb.net/hijarvis?retryWrites=true', {useNewUrlParser: true})
```
DB_PROD=mongodb+srv://admin:root@cluster0-r5gmo.gcp.mongodb.net/hijarvis?retryWrites=true